### DISCUSSION TITLE
Upgrade fail with  ImageVerificationFailed

### DISCUSSION THREAD

**Discussion by gerijani** [okd-project/okd#1025] [Q&A] [answered]
After scaling up the cluster from one master node to a 3 master, 6 worker and removed 

useUnsupportedUnsafeNonHANonProductionUnstableEtcd: *
useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer: *

set by the installer we get this message by trying upgrade to 4.7.0-0.okd-2021-07-03-190901 from 4.6.0-0.okd-2021-02-14-205305

ImageVerificationFailed The update cannot be verified: unable to locate a valid signature for one or more sources
ImageVerificationFailed Unable to apply 4.7.0-0.okd-2021-07-03-190901: the image may not be safe to use

All CO are available and there is no degraded. MCPs also do not show degraded state

**Comment by vrutkovs** [comment, marked as answer]
#605

**Comment by gerijani** [comment]
Thanks!
